src/index.js

```diff
@@ -0,0 +1,145 @@
+let globalTemp;
+const cityInput = document.getElementById('name');
+const cityName = document.getElementById('city-name');
+const skyOptions = document.getElementById('skyOptions');
+const sky = document.getElementById('skyEmojis');
+
+//Wave 1
+const updateTemp = (temp) => {
+  const tempValueContainer = document.getElementById('tempValue');
+  tempValueContainer.textContent = temp;
+  globalTemp = temp;
+  updateTempColor(temp);
+  changeLandscape(temp);
+};
+
+const increaseTemp = () => {
+  globalTemp += 1;
+  updateTemp(globalTemp);
+};
+
+const decreaseTemp = () => {
+  globalTemp -= 1;
+  updateTemp(globalTemp);
+};
+
+const changeLandscape = (temp) => {
+  const landscape = document.getElementById('landscape');
+  let ground = '🥶__🧤_🏂_🧊__🧊_';
+  if (temp >= 80) {
+    ground = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp >= 70) {
+    ground = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp >= 60) {
+    ground = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (temp >= 50) {
+    ground = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+  landscape.textContent = ground;
+};
+
+const updateTempColor = (temp) => {
+  const tempValueColor = document.getElementById('tempValue');
+  let color = 'cold';
+  if (temp >= 80) {
+    color = 'hot';
+  } else if (temp >= 70) {
+    color = 'warm';
+  } else if (temp >= 60) {
+    color = 'cozy';
+  } else if (temp >= 50) {
+    color = 'cool';
+  }
+  tempValueColor.className = color;
+};
+
+//Wave 3
+const updateCity = () => {
+  cityName.textContent = cityInput.value;
+};
+
+//Wave 4
+const getLatLong = () => {
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: cityInput.value,
+        format: JSON,
+      },
+    })
+    .then((locationData) => {
+      const lat = locationData.data[0].lat;
+      const lon = locationData.data[0].lon;
+      getWeather(lat, lon);
+    });
+};
```

Comment:
This works great! As an alternative, can you consider what this would look like with promise chaining? Also, make sure to add a `catch` to handle failed requests.